bom/pom.xml

```diff
@@ -90,7 +93,7 @@
     <version.commons-logging>1.2</version.commons-logging>
     <version.netty>4.1.24.Final</version.netty>
 
-    <version.mp-config>1.2.1</version.mp-config>
+    <version.mp-config>1.3</version.mp-config>
```

Comment:
Just wondering, do we need the MP Config API in the build as well as SmallRye Config?

Reply:
Nope, in fact we don't. I removed the <dependency> but missed the property, I think.

Reply:
We do need the `version.mp-config` to reference the TCK, and in that case, I think keeping the TCK and API in sync within the context of Thorntail might be smart?

Reply:
is there a way to ensure one doesn't get updated without the other?

trying to minimize moving pieces

Reply:
I think at this point, not easily. We can keep our API + TCK in sync, but then SmallRye might vary.  We can keep SmallRye + API in sync, but then TCK might vary.